packages/plugin-solana-agentkit/package.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "@elizaos/plugin-solana-agentkit",
+    "version": "0.1.7-alpha.1",
+    "main": "dist/index.js",
+    "type": "module",
+    "types": "dist/index.d.ts",
+    "dependencies": {
+        "@coral-xyz/anchor": "0.30.1",
+        "@elizaos/core": "workspace:*",
+        "@elizaos/plugin-tee": "workspace:*",
+        "@elizaos/plugin-trustdb": "workspace:*",
```

Comment:
I wonder if the root turbo config is up to date

Reply:
@odilitime hi sorry, i dismiss your approve again lol, was trying to rebase it to fix the conflicts.